src/test/e2e/tests/selectAzureTenant.test.ts

```diff
@@ -1,10 +1,15 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import { test } from '../baseTest';
+
+import { expect, test } from '../baseTest';
+import { APICenter, TestENV, Timeout, VSCode } from '../utils/constants';
+import VscodeOperator from '../utils/vscodeOperator';
+
 test('select Tenant', async ({ workbox }) => {
```

Comment:
So this test case is not 'select Tenant' now? 
It is showing subs when there is only 1 tenant

Reply:
Indeed it is. The step of selecting tenant is put in prepare step, and this case becomes to verify whether subs meets expectations. Can I change title to "showing subs"？

Reply:
> So this test case is not 'select Tenant' now? It is showing subs when there is only 1 tenant

change case name to "validate subscription"